### ISSUE TITLE
Dock window is not avaliable on Linux Mint Cinammon 

### ISSUE THREAD

**Issue by RuanLucasGD** [FlaxEngine/FlaxEngine#1708]
**Issue description:**
I cannot set as full screen the window and cannot dock on sides.

**Steps to reproduce:**
Open a window and try dock

https://github.com/FlaxEngine/FlaxEngine/assets/79365912/086207c0-6c65-4f51-a02f-0b069b723d83

**Flax version:**
1.6 master

**Comment by MineBill** [comment]
Distro/DE? Docking/maximizing works perfectly fine under KDE, both X11/Wayland.

**Comment by RuanLucasGD** [comment]
@MineBill Mint Cinnamon

**Comment by nothingTVatYT** [comment]
Here on KDE Plasma desktop the docking does not work. What is suspicious is that at first it seems to start to react if you drag the custom title bar (gets blue) but stops as soon as the cursor leaves the window to be dragged. That could mean it's a mouse-leave or lost-focus event handling problem.

**Comment by MineBill** [comment]
@nothingTVatYT Could you share what version of KDE you are running? I find it baffling that it works on my install but not on yours.

**Comment by nothingTVatYT** [comment]
It is the Garuda Arch Linux Dragonized KDE Edition.
About this system reports:
```
Operating System: Garuda Linux 
KDE Plasma Version: 5.27.8
KDE Frameworks Version: 5.110.0
Qt Version: 5.15.11
Kernel Version: 6.5.7-zen1-1-zen (64-bit)
Graphics Platform: X11
Processors: 24 × AMD Ryzen 9 3900X 12-Core Processor
Memory: 31,2 GiB of RAM
Graphics Processor: NVIDIA GeForce RTX 4060 Ti/PCIe/SSE2
```

**Comment by MineBill** [comment]
Just to make sure, here is "docking" working for me:

https://github.com/FlaxEngine/FlaxEngine/assets/30367251/50f3b569-6300-4b50-8263-87bbc3a92a3b

Is this the behavior you cannot replicate?

**Comment by nothingTVatYT** [comment]
What that shows is not the docking done by the Flax Editor but a window arrangement done by the window manager.
The docking functionality we're talking about is pulling a floating window (undocked) back into the main editor window.

**Comment by MineBill** [comment]
Yes, that kind of docking is broken but @RuanLucasGD demonstrates that they can't get their window manager to perform the kind of docking my video shows, like maximizing the window or docking it to the side.

**Comment by nothingTVatYT** [comment]
It looks like he dragged it by the root title bar where the undocking is supposed to work on the custom (blue) title bar. But I think he tries to pull the sub-window back into the main window. [RuanLucasGD](https://github.com/RuanLucasGD), can you clarify?

**Comment by RuanLucasGD** [comment]
@nothingTVatYT @MineBill the window docking is broken for me, after open some windows i cannot open any asset and the dock indicator doesn't hiden

https://github.com/FlaxEngine/FlaxEngine/assets/79365912/6d8497ac-a6cf-49de-b275-884b78ad0972

**Comment by RuanLucasGD** [comment]
#2020 fix part of this issue

**Comment by nothingTVatYT** [comment]
Could you please record and describe in detail what is the remaining problem? I'm not sure I understand it correctly and something I can reproduce would be a tremendous help.

**Comment by RuanLucasGD** [comment]
@nothingTVatYT you can see on the first video the "maximize" button don't exist on another windows, only on main editor window. 
![image](https://github.com/FlaxEngine/FlaxEngine/assets/79365912/d6cdb008-9208-4cc4-a20f-a91974633e62)

https://github.com/FlaxEngine/FlaxEngine/blob/72f45afa4505e2c4a5df521dbe3b74edf45612ad/Source/Engine/Platform/Linux/LinuxWindow.cpp#L121C39-L121C39

Hey @nothingTVatYT thanks for improving Linux support :)

**Comment by nothingTVatYT** [comment]
It works in KDE so my guess is that there are some WM hints missing and KDE simply ignores that whereas Cinnamon won't show the icons. I'm going to need a Mint installation to verify.

**Comment by RuanLucasGD** [comment]
@nothingTVatYT maybe only install cinnamon is necessary
